### PR TITLE
Icons

### PR DESCRIPTION
**Pull request by Voros2**
Rather than using the pixelated assets of Freedoom for the icons, I put together these not-so-great icons. They can serve as suitable placeholders. Addresses https://github.com/freedoom/freedoom/issues/123

### CODE REVIEW

File: CREDITS [hunk 1]
```diff
@@ -409,9 +409,9 @@ S: dolorous
 E: pooka109@cox.net
 D: sounds
 
-S: catoptromancy
+S: Catoptromancy
 E: catoptromancy@yahoo.com
-D: Levels
+D: levels, playtesting, bugfixes, textures
```
Comment (fragglet) [inline CREDITS]: Can you fix this please?

Comment (Voros2) [inline CREDITS]: Fix what?

**Comment by Voros2** [comment]
I mixed up the file names, so now it should be corrected.

**Comment by chungy** [comment]
It'd be nicer if an actual logo were made for Phases 1 and 2. I don't really like these at present compared to the reuse of in-game sprites.

**Comment by Voros2** [comment]
Suggestions?

**Comment by fragglet** [comment]
Same problem here - you still have unrelated changes to textures, CREDITS, buildcfg.txt, etc. If you can fix these then I'll merge.

**Pull request closed without merging**